packages/py-api/py_api/utilities/logging.py

```diff
@@ -12,7 +12,7 @@
     'formatters': {
         'logformatter': {
             'format': '[%(asctime)s][%(levelname)s][%(name)s]: %(message)s',
-            'datefmt': '%H:%M',
+            'datefmt': '%Y-%m-%d %H:%M:%S',

```

Comment:
Do you think having seconds here will provide any significant value?

Reply:
Not really. I mean in incident reports it is often given only HH:MM.